var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -860,6 +860,7 @@ def cmake_args(self):
                     define("CUDA_SDK_ROOT_DIR", "IGNORE"),
                     define("CUDA_NVCC_EXECUTABLE", "IGNORE"),
                     define("LIBOMPTARGET_DEP_CUDA_DRIVER_LIBRARIES", "IGNORE"),
+                    define("LIBOMPTARGET_BUILD_CUDA_PLUGIN", "FALSE"),
```

Comment:
```suggestion
                    define("LIBOMPTARGET_BUILD_CUDA_PLUGIN", False),
```

Reply:
Using a bool here instead of the string 'FALSE' results in the cmake argument being formatted as:
```
'-' 'D' 'L' 'I' 'B' 'O' 'M' 'P' 'T' 'A' 'R' 'G' 'E' 'T' '_' 'B' 'U' 'I' 'L' 'D' '_' 'C' 'U' 'D' 'A' '_' 'P' 'L' 'U' 'G' 'I' 'N' ':' 'B' 'O' 'O' 'L' '=' 'O' 'F' 'F'
```